src/redis.h

```diff
@@ -927,8 +928,12 @@ struct redisCommand {
     int lastkey;  /* The last argument that's a key */
     int keystep;  /* The step between first and last key */
     long long microseconds, calls;
+    sampleReservoire samples;
 };
 
+#define CMDTIME_PERCENTILES_COUNT 5
+const static double CMDTIME_PERCENTILES[] = {.25, .50, .90, .99, .999};
```

Comment:
Or even better. Create a compiler assertion (if not already available):
```
#define CASSERT(expr) extern void compiler_assert__(char[(expr)?1:-1])
```

In the header:
```
extern const double CMDTIME_PERCENTILES[];
```

Then in the translation unit:
```
const double CMDTIME_PERCENTILES[] = {...};
CASSERT(sizeof(CMDTIME_PERCENTILES)/sizeof(*CMDTIME_PERCENTILES) == CMDTIME_PERCENTILES_COUNT);
```

So the compiler will blow up if everything isn't synchronized.